### DISCUSSION TITLE
Is it possible to set a custom drum map as the default?

### DISCUSSION THREAD

**Discussion by chunga2020** [musescore/MuseScore#19006] [Q&A]
As of MuseScore 4.1.1, it is possible to specify a default style through `Preferences > Score > Style`.  Is there something similar for setting a default drum map?  I've customized mine so that all toms are on unique spaces, whereas in the stock drum map, certain spaces are shared by two toms.

I've looked through the settings, and through the documentation, but can find no such option.

**Comment by MarcSabatella** [comment]
Not currently, but you can set up a score the way you like and save it to your Templates folder, then just use that when creating new scores.  Aklso, as you've probably seen, you can save and load drumset definitions.

**Reply by chunga2020** [reply]
Thanks for the response.  I hadn't really looked into what templates were, precisely.  I suppose they'll work well enough.  And yes, I did save my customized drum map definitions.
